src/molecules/Snackbar/Snackbar.native.js

```diff
@@ -111,59 +115,61 @@ const Snackbar = ({ variant, title, action, onClose, maxLines, icon, position })
         }}
         onLayout={handleLayout}
       >
-        <Size width={`${SNACKBAR_WIDTH}px`}>
-          <Space padding={[1.5]}>
-            <Flex flexDirection="row" alignItems="center">
-              <SnackbarContainer variant={variant}>
-                {icon ? (
-                  <Space padding={[0, 1, 0, 0]}>
-                    <View>
-                      <Icon name={icon} size="medium" fill="light.900" testID="snackbar-icon" />
-                    </View>
-                  </Space>
-                ) : null}
-                <Space padding={[0, 2, 0, 0]}>
-                  <Flex flex={1}>
-                    <View>
-                      <Text size="medium" color="light.900" maxLines={maxLines}>
-                        {title}
-                      </Text>
-                    </View>
-                  </Flex>
-                </Space>
-                {action?.label ? (
-                  <Space padding={[0, 0.75, 0, 0]}>
-                    <View>
-                      <Button
-                        variant="secondary"
-                        size="xsmall"
-                        variantColor="light"
-                        onClick={handleAction}
-                        testID="ds-snackbar-action-button"
-                      >
-                        {action.label}
-                      </Button>
-                    </View>
+        {isVisible || isContentVisible ? (
```

Comment:
`isContentVisible` alone should be enough? because here also `isVisible` will hide the snackbar immediately?

Reply:
- Only `isContentVisible` is not working in one case when app launched & need to show snackbar(for example if server is not connected)
- `isVisible || isContentVisible` will only hide content once both will be false.

I will add a video here for reference 👍

Reply:
https://user-images.githubusercontent.com/30696104/116347619-2d6a1b80-a80a-11eb-8ebd-e130add5737f.mp4